feedTheMonster.ts

```diff
@@ -212,6 +212,7 @@ class App {
 
   private handleLoadingScreen = () => {
     if (this.is_cached.get(lang)) {
+      this.logDownloadPercentageComplete(100,0);
```

Comment:
Why we are logging a download percentage complete if the language content is already cached, @amitsinghsutara ? 🤔